test/bench/etc/run_granular_benchmarks.js

```diff
@@ -18,7 +19,7 @@ const DOCUMENT_ROOT = path.resolve(`${__dirname}/../documents`);
   // FIXME(NODE-5759): replace this with a call to Package.install()
   await new Task({
     documentPath: path.resolve(`${DOCUMENT_ROOT}/binary_small.json`),
-    library: `bson#main`,
+    library: LIBRARY_SPEC,
```

Comment:
Works for me, but is there a way we can remove this task from the final output since it isn't "real" in a way

Reply:
It isn't a part of the output. This task runs and fails in CI since the targeted library isn't yet installed when run on a fresh system. Even if it were to run to completion, the results are not collected or written.

Reply:
All of the actual benchmarks are run and have results collected and written using the `runSuiteAndWriteResults` helper in `test/bench/granular/common.ts`.

Reply:
My mistake, I thought I was seeing small binary twice, it was with/without promoteBuffers